packages/wmr/test/production.test.js

```diff
@@ -889,6 +889,31 @@ describe('production', () => {
 			const index = await fs.readFile(indexHtml, 'utf8');
 			expect(index).toMatch(/# hello world/);
 		});
+
+		it('should not support fetching resources from external link prerender', async () => {
+			await loadFixture('prerender-external-resource-fetch', env);
+			instance = await runWmr(env.tmp.path, 'build', '--prerender');
+			const code = await instance.done;
+
+			const nodeVersion = process.version;
+			const isNodeVersionOver18 = Number(nodeVersion.split('.')[0].slice(1)) >= 18;
+			if (isNodeVersionOver18) {
```

Comment:
Is there a way to run tests on multiple versions?

Reply:
Oh, and two versions, 12 and 14, are running now. This felt like a no-brainer in the future.

Reply:
I'm also considering breaking down this test.

Instead of branching using `if` in one test, write a test for Node.js version 17 or less and a test for Node.js version 18 or more, and skip if the version is not applicable. I think it is also possible.
This part is one of the things I want to discuss in this PR.

Reply:
> Instead of branching using if in one test, write a test for Node.js version 17 or less and a test for Node.js version 18 or more, and skip if the version is not applicable.

I like this idea. There's slightly different behavior between the two that (ideally) we'd test.

We can definitely alter the CI config to cover newer versions of Node.

Reply:
I completely forgot about this.
Last time I called fetch without the leading slash here, but this test does not work with the regex changes you reviewed.
https://github.com/preactjs/wmr/blob/main/packages/wmr/test/fixtures/prerender-resource-fetch/index.js#L2

I think to need to modify the test here as well. I think the comments here will be helpful for that.
https://github.com/preactjs/wmr/pull/935#discussion_r977168334

Reply:
Oops, no correction of this was necessary.